source/quick_start_guide/index.html.md.erb

```diff
@@ -9,10 +9,10 @@ GOV.UK Pay is based on REST principles with endpoints returning data in JSON
 format, and standard HTTP error response codes. The platform API allows you
 to:
 
-- begin and complete payments
-- issue refunds
-- view the event history for individual payments
-- view transactions within a specified time period
+* begin and complete payments
```

Comment:
Should we be linking to individual sections of the docs here or should this remain an introduction?